src/main/java/org/isf/medicalinventory/gui/InventoryWardBrowser.java

```diff
@@ -48,343 +52,565 @@
-            }
-        }
-        return stateComboBox;
-    }
-
-    private JLabel getStateLabel() {
-        if (stateLabel == null) {
-            stateLabel = new JLabel(MessageBundle.getMessage("angal.inventory.state.label"));
-            stateLabel.setHorizontalAlignment(SwingConstants.RIGHT);
-        }
-        return stateLabel;
-    }
-
-    @Override
-    public void InventoryInserted(AWTEvent e) {
-        if (inventoryList != null) {
-            inventoryList.clear();
-        }
-        jTableInventory.setModel(new InventoryBrowsingModel());
-    }
-
-    @Override
-    public void InventoryUpdated(AWTEvent e) {
-        if (inventoryList != null) {
-            inventoryList.clear();
-        }
-        jTableInventory.setModel(new InventoryBrowsingModel());
-    }
-
-    @Override
-    public void InventoryCancelled(AWTEvent e) {
-        if (inventoryList != null) {
-            inventoryList.clear();
-        }
-        jTableInventory.setModel(new InventoryBrowsingModel());
-    }
-}
\ No newline at end of file
+	private static final long serialVersionUID = 1L;
+	private static int PAGE_SIZE = 50;
+	JButton next;
+	JButton previous;
+	JComboBox<Integer> pagesCombo = new JComboBox<>();
+	JLabel under = new JLabel("/ 0 " + MessageBundle.getMessage("angal.inventory.page.text"));
+	private GoodDateChooser jCalendarTo;
+	private GoodDateChooser jCalendarFrom;
+	private LocalDateTime dateFrom = TimeTools.getDateToday0();
+	private LocalDateTime dateTo = TimeTools.getDateToday24();
+	private JLabel jLabelTo;
+	private JLabel jLabelFrom;
+	private JPanel panelHeader;
+	private JPanel panelFooter;
+	private JPanel panelContent;
+	private JButton closeButton;
+	private JButton newButton;
+	private JButton editButton;
+	private JButton deleteButton;
+	private JButton viewButton;
+	private JScrollPane scrollPaneInventory;
+	private JTable jTableInventory;
+	private String[] pColums = {
+			MessageBundle.getMessage("angal.inventory.referenceshow.col").toUpperCase(),
+			MessageBundle.getMessage("angal.common.ward.col").toUpperCase(),
+			MessageBundle.getMessage("angal.common.date.col").toUpperCase(),
+			MessageBundle.getMessage("angal.inventory.status.col").toUpperCase(),
+			MessageBundle.getMessage("angal.common.user.col").toUpperCase()
+	};
+	private int[] pColumwidth = { 150, 150, 100, 100, 150 };
+	private JComboBox<String> statusComboBox;
+	private JLabel statusLabel;
+	private int startIndex;
+	private int totalRows;
+	private MedicalInventoryManager medicalInventoryManager = Context.getApplicationContext().getBean(MedicalInventoryManager.class);
+	private WardBrowserManager wardBrowserManager = Context.getApplicationContext().getBean(WardBrowserManager.class);
+	private List<MedicalInventory> inventoryList;
+
+	public InventoryWardBrowser() {
+		initComponents();
+	}
+
+	private void initComponents() {
+
+		setDefaultCloseOperation(JFrame.DISPOSE_ON_CLOSE);
```

Comment:
You have managed to get the formatting consistent but it is still wrong.    The indention here is 8 spaces and the project standards are 4.   Please use the formatting guides provided in the `.ide-settings` directory.  Thanks.